lib/sql.reserved.js

```diff
@@ -0,0 +1,764 @@
+'use strict';
+
+// A list of keywords recieved from:
+// https://www.postgresql.org/docs/current/static/sql-keywords-appendix.html
+
+module.exports = [
```

Comment:
We can leave in this list only reserved words from first column in https://www.postgresql.org/docs/current/static/sql-keywords-appendix.html